src/Illuminate/Notifications/Message.php

```diff
@@ -0,0 +1,225 @@
+    }
+
+    /**
+     * Indicate that the notification gives information about a successful operation.
+     *
+     * @return $this
+     */
+    public function success()
+    {
+        return $this->level('success');
+    }
+
+    /**
+     * Indicate that the notification gives information about an error.
+     *
+     * @return $this
+     */
+    public function error()
+    {
+        return $this->level('error');
+    }
+
+    /**
+     * Set the "level" of the notification (success, error, etc.).
+     *
+     * @param  string  $level
+     * @return $this
+     */
+    public function level($level)
+    {
+        $this->notification->level = $level;
+
+        return $this;
+    }
+
+    /**
+     * Set the subject of the notification.
+     *
+     * @param  string  $subject
+     * @return $this
+     */
+    public function subject($subject)
+    {
+        $this->subject = $subject;
+
+        return $this;
+    }
+
+    /**
+     * Add a line of text to the notification.
+     *
+     * @param  \Illuminate\Notifications\Action|string  $line
+     * @return $this
+     */
+    public function line($line)
+    {
+        return $this->with($line);
+    }
+
+    /**
+     * Add a line of text to the notification.
+     *
+     * @param  \Illuminate\Notifications\Action|string|array  $line
+     * @return $this
+     */
+    public function with($line)
+    {
+        if ($line instanceof Action) {
+            $this->action($line->text, $line->url);
+        } elseif (! $this->actionText) {
+            $this->introLines[] = $this->formatLine($line);
+        } else {
+            $this->outroLines[] = $this->formatLine($line);
+        }
+
+        return $this;
+    }
+
+    /**
+     * Format the given line of text.
+     *
+     * @param  string|array  $line
+     * @return string
+     */
+    protected function formatLine($line)
+    {
+        if (is_array($line)) {
+            return implode(' ', array_map('trim', $line));
+        }
+
+        return trim(implode(' ', array_map('trim', explode(PHP_EOL, $line))));
+    }
+
+    /**
+     * Set the notification's options.
+     *
+     * @param  array  $options
+     * @return $this
+     */
+    public function options(array $options)
+    {
+        $this->options = $options;
+
+        return $this;
+    }
+
+    /**
+     * Configure the "call to action" button.
+     *
+     * @param  string  $text
+     * @param  string  $url
+     * @return $this
+     */
+    public function action($text, $url)
+    {
+        $this->actionText = $text;
+        $this->actionUrl = $url;
+
+        return $this;
+    }
+
+    /**
+     * Get the instance as an array.
+     *
+     * @return array
+     */
+    public function toArray()
+    {
+        return [
+            'notifiable' => $this->notifiable,
+            'application' => $this->notification->application,
+            'logoUrl' => $this->notification->logoUrl,
+            'level' => $this->notification->level,
```

Comment:
`application`, `logoUrl` and `level` most likely going to be shared for a single Notification. so might as well do this.